.github/workflows/specimin_evaluation_CI.yml

```diff
@@ -0,0 +1,204 @@
+        run: |
+         set -ex
+         mkdir -p $(pwd)/ASHE/CI_REPO_CLONE_SPACE
+         chmod 777 $(pwd)/ASHE/CI_REPO_CLONE_SPACE
+         ls -ld $(pwd)/ASHE/CI_REPO_CLONE_SPACE
+          
+      - name: Verify example.properties exists
+        run: |
+          set -ex
+          if [ -f ASHE/src/main/resources/example.properties ]; then 
+            echo "example.properties found"
+          else 
+            echo "example.properties not found"
+            exit 1
+          fi
+
+      - name: Rename example.properties to config.properties file
+        run: |
+          set -ex
+          mv ASHE/src/main/resources/example.properties ASHE/src/main/resources/config.properties
+          if [ -f ASHE/src/main/resources/config.properties ]; then 
+            echo "config.properties created"
+          else 
+            echo "config.properties not created"
+            exit 1
+          fi
+
+      - name: Give write permissions to config.properties
+        run: |
+          set -ex
+          chmod +w ASHE/src/main/resources/config.properties
+          ls -l ASHE/src/main/resources/config.properties
+
+      - name: Update ASHE Config File to update SPECIMIN path
+        run: |
+          set -ex
+          chmod +w ASHE/src/main/resources/config.properties
+          # Update the specimin.tool.path key with the new value
+          sed -i 's|^specimin.tool.path=.*|specimin.tool.path='$(pwd)'|' ASHE/src/main/resources/config.properties
+          # Display the updated config.properties file for verification
+          cat ASHE/src/main/resources/config.properties
+
+      - name: Display updated config.properties
+        run: |
+          set -ex
+          cat ASHE/src/main/resources/config.properties
+
+      - name: Make all scripts under ashe_scripts executable
+        run: |
+          set -ex
+          chmod +x ashe_scripts/*.py
+
+      - name: List Files in ashe_scripts for Debugging
+        run: |
+          set -ex
+          ls -l ashe_scripts
+
+      - name: Run the script
+        run: |
+          set -ex
+          python3 ashe_scripts/run_ashe_for_stats.py \
+            $(pwd)/ASHE \
+            $(pwd)/CI_repository_list.csv \
+            $(pwd)/ASHE/CI_REPO_CLONE_SPACE \
+            $(pwd)/ASHE/src/main/resources/config.properties
+            
+      - name: Parse accuracy percentage
+        id: parse_accuracy_percentage
+        run: |
+          set -ex
+          grep 'Fully successful from minimization to compilation' $(pwd)/ASHE/logs/specimin_statistics.txt | awk '{print $NF}' > current_run_accuracy_percentage.txt
+          cat current_run_accuracy_percentage.txt
+
+      - name: Read and update evaluation accuracy
+        id: read_and_update
+        run: |
+          set -ex
+          current_accuracy=$(cat current_run_accuracy_percentage.txt)
+          echo "Current accuracy: $current_accuracy"
+          previous_run_accuracy=${{ secrets.LATEST_SPECIMIN_EVAL_PERCENTAGE }}
```

Comment:
I'm concerned that this is a single global variable for all pull requests. Consider the following scenario in the future:
* on the `main` branch, the last CI run produced 90% accuracy
* from the `feature-1` branch, developer A opens a PR with 85% accuracy. This CI task fails, because 85 < 90 (all good so far!)
* from the `feature-2` branch, developer B opens another PR with 87% accuracy. Will this CI task pass? I think it will, because the latest run will be the run of `feature-1` that failed with 85% accuracy, and 87 > 85. 

So, I think that this really needs to only be updated if the current branch is `main`: otherwise, there could be confusion between PRs.